docs/pages/guides/new-architecture.mdx

```diff
@@ -22,14 +22,14 @@ To learn more about the ideas and motivations behind the New Architecture, we re
 <BoxLink
   title="Introduction to the New Architecture"
   description="A starting point for exploring the various topics related to the New Architecture."
-  href="https://reactnative.dev/docs/the-new-architecture/landing-page"
+  href="https://github.com/reactwg/react-native-new-architecture#guides"
```

Comment:
```suggestion
  href="https://reactnative.dev/docs/the-new-architecture/landing-page"
```

This is intended here to point toward the landing page of the New architecture in React Native docs.